build.xml

```diff
@@ -1381,7 +1380,6 @@
             **/*.json
             **/*.jsp
             **/*.jspf
-            **/*.lcf
             **/*.md
             **/*.php
             **/*.pl
```

Comment:
There are additional mentions of default.lcf in the release-build targets: package-windows, package-macosx and package-linux

Reply:
Thanks for spotting Bob, I'd changed locally however missed on upload.